lib/rides/pages/search_suggestion_page.dart

```diff
@@ -108,10 +108,16 @@ class _SearchSuggestionPage extends State<SearchSuggestionPage> {
 
   //todo: get possible Rides from Algorithm
   void loadRides() async {
+    int riderid = SupabaseManager.getCurrentProfile()?.id ?? -1;
     List<dynamic> data = await supabaseClient
         .from('drives')
         .select('*, driver:driver_id (*)')
         .eq('start', _startController.text)
+        .eq('end', _destinationController.text)
```

Comment:
let's remove all these filters, because they are present in Simon's PR?

Reply:
I mean this is basically an algorithm change and not in my PR, but I don't see why it's necessary because it completely removes the possibility of partial rides